spec/features/apply_to_du_spec.rb

```diff
@@ -14,6 +14,7 @@
 
     fill_in "Twitter username", with: "@beepboopbeep"
     fill_in "Blog URL", with: "http://blog.awesome.com"
+    fill_in "Pronouns", with: "They/Them"
```

Comment:
@lilliealbert thank you for the review! I incorporated the changes into the same commit.